src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -509,40 +509,63 @@ public String getAudioCodec() {
 	@Override
 	public String toString() {
 		StringBuilder result = new StringBuilder();
-		result.append("id: ");
-		result.append(getId());
-		result.append(", lang: ");
-		result.append(getLang());
+		if (!getLang().equals("und")) {
+			result.append("Id: ").append(getId());
```

Comment:
@Nadahar Yes, believe me, i well tested it, and i see that you commit withdrawed all the language code with id=0 from the log, and believe me it was many testing files.
"und" is only set by our code, when no language is found in the audio file, so i didn't want factice informations for debugging...

About the extra language code meaning, i don't need it for debugging purpose, the id in the file is suffisant to check in the other part of the log, IMHO.